### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by allun26**

### CODE REVIEW

**Comment by mosaaleb** [comment]
Reviewing

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+   <main>
+        <nav>
+            <ul>
+                <li><a href="#"><div></div></a></li>
+                <span></span>
+                <li><a href="#">Store</a></li>
```
Comment (mosaaleb) [inline index.html]: It's better for the apple logo to be set as `` tag [check here](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                <span></span>
+                <li><a href="#">Mac</a></li>
+                <span></span>
+                <li><a href="#">iPod</a></li>
```
Comment (mosaaleb) [inline index.html]: What are these `span` for? If it is for the line separator between the links, you can add its styles inside the `<li>` styles itself as `border-right` without the need to repeat yourself.
Also if you want to exclude the last element (search item) from adding `border-right` you can use [`not(:last-child)`](https://stackoverflow.com/questions/7272302/excluding-last-child?noredirect=1&lq=1) css selector

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                <li><a href="#">Support</a></li>
+                <span></span>
+                <li><input type="text"></li>
+            </ul>
+        </nav>
+       <div class="top">
+            <section>
+                <img class="imgH1" src="images/your_verse_title_small_2x.png" alt="title">
+                <img class="imgH2"src="images/your_verse_subhead_small_2x.png" alt="text">
+                <div class=headerLink>
```
Comment (mosaaleb) [inline index.html]: The images `imgH1`, `imgH2` are also links

**Review by mosaaleb — CHANGES_REQUESTED** [review]
### Hi @allun26 :wave:

Well done :clap:. You have met most of the requirements, yet there some minor requirements need to be handled before moving on.

Please see my comments below

**Also** there is a horizontal scroll bar appearing in the browser. I have inspected the layout and the `bottom` section having width more than 100%. Let's fix it.

Please submit a new code review when you done with these requirements.

### Happy Coding :computer:

**Comment by samgaco** [comment]
### Hi @allun26 :vulcan_salute:  everything perfect but:

* The horizontal scrolling bar still showing.
* The four images right above the footer are not showing. They should be set as backgrounds for `<div>` elements wrapped within `<a>` tags;

background

### Submit back for another code review when you get these things done :muscle:

**Comment by Oghenebrume50** [comment]
Good work @allun26 you did all the basic things right, as of now the horizontal scroll bar still shows but I am sure you will learn more about that in your next project please, checkout [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) and use it for your next HTML projects, for now, you can submit this and move on :+1:

**Pull request merged**